lib/acpi/acpi.c

```diff
@@ -52,24 +52,6 @@ static void notify_handler(ACPI_HANDLE device, UINT32 value, void *ctx)
 	ACPI_INFO(("Received a notify 0x%X", value));
 }
 
-static ACPI_STATUS region_handler(UINT32 Function, ACPI_PHYSICAL_ADDRESS address, UINT32 bit_width,
```

Comment:
small typo in the commit message: s/out/our

Reply:
> small typo in the commit message: s/out/our

thanks, fixed